tests/test_clinvar_record.py

```diff
@@ -0,0 +1,293 @@
+from datetime import datetime
+import json
+import unittest
+
+from eva_cttv_pipeline import clinvar_record
+from eva_cttv_pipeline import utilities
+
+
+class TestClinvarRecord(unittest.TestCase):
+    def setUp(self):
```

Comment:
Change to setUpClass for function to be run just once (repeat for other relevant classes)